Makefile

```diff
@@ -96,17 +96,17 @@ test/int/run-api-s3:
 ####
 ci/integration/tests/docker-compose-up:
 	export DOCKER_BUILDKIT=1
-	docker compose -f ./tests/integration/docker-compose-ci-integration.yml up --build --detach
+	docker compose -f ./tests/integration/docker-compose.yml -f ./tests/integration/docker-compose.ci.yml up --build --detach
```

Comment:
move all common stuff to the base docker-compose file. use the `.ci.` file to just made the 1 change for CI.  see more below.